calcom/cal.com
# feat: Adds new v2 design for app detail page. Fixes #3779

## What does this PR do?

Implements new design for app detail page. 

Screenshots of actual work: 
![image](https://user-images.githubusercontent.com/2969573/187030610-5dd7f16a-91dd-46b7-82a9-3f24d4e9efe7.png)
<img width="368" alt="Screenshot 2022-08-27 at 14 44 14" src="https://user-images.githubusercontent.com/2969573/187030771-c64deb9a-17b2-4981-be22-160849f64067.png">


Fixes #3779
 Loom Video: https://www.loom.com/share/13288fbc55b04b9287094deae378bda6

**Environment**: Staging(main branch) / Production

## Type of change


## How should this be tested?

Visit app detail pages and ensure layout is correct. Also verify v1 page is NOT impacted by this change. 

## Open questions

1. @Jaibles  Let me know what you think about the mobile version :) And of course any other design feedback is welcome! 
2. During the Loom video I also resize my browser to a very small height. Currently I'm using the default CSS `sticky`. Its default behaviour however is that you only see the out of view portion once you've reached the bottom of the page. I'd rather have it immediately scroll down, and then stay sticky at the bottom. However, in order to do this, we need to either write quite some JS, or implement a fixed height and scrollbar in the sticky section on the right. Both don't really have my preference, since the use case for this is pretty small (so adding JS might not make sense), and adding a fixed height + scroll bar makes the UX worse. What do you guys think?
